data/data.json

```diff
@@ -15,7 +15,8 @@
       ],
       "gdcc_member": false,
       "doi_authority": "\u2415",
-      "contact_email": "abacus.support@ubc.ca"
+      "contact_email": "abacus.support@ubc.ca",
+      "dv_hub_id": "01893C8A-2503-4E23-A873-59CE9C40D568"
```

Comment:
@jggautier as discussed [in Slack](https://iqss.slack.com/archives/C07CFTZ0YG6/p1753207878591389?thread_ts=1753111648.043439&cid=C07CFTZ0YG6), there's a new "dv_hub_id" column in the [spreadsheet](https://docs.google.com/spreadsheets/d/1bfsw7gnHlHerLXuk7YprUT68liHfcaMxs1rFciA-mEo/edit?usp=sharing) that needs to be manually populated for each new installation we add. Do you want anything about this in the README? Or is it obvious enough from context (and oral tradition, I guess) that this field should be filled in?

Reply:
Yeah there are a few undocumented things about this spreadsheet that always make me uneasy about others editing it. So I wouldn't mind adding something to the readme about this new column. I can add something about it to this PR a bit today. Is that okay with you and @jp-tosca?

Could one of us also make the new column the new column A or B, so it's easier to see when adding installations?

Reply:
Hmmm, @jp-tosca it looks like you're adding things in the readme about that ID column. So I won't.

I'll wait to hear back from you and @pdurbin if y'all think it's okay to move that column to column A or B so it's easier to see when adding installations.